### DISCUSSION TITLE
wider "reading pane"

### DISCUSSION THREAD

**Discussion by eosfor** [Lednerb/bilberry-hugo-theme#607] [Ideas]
Hello @igor-baiborodine,

Is it possible somehow to make the "reading pane" wider? on wide screens it really looks too narrow, IMHO. Can i do it myself somehow? i'm not a web dev, though, so it may need some more careful explanation if it is possible to do it without chnagenig the main template :D

**Comment by Sp-cy** [comment]
Hello, this is an open source project so all explanations are in the documentation which you must carefully read. There is this variable in v4/assets/sass/them.scss: 
`// Width of the main content area where the the actual content's width is $site-width - 100px
$site-width: {{ .Param "siteWidth" | default "800px" }};`

You could modify it adding the variable to your site config's Params like explained in the following section of the theme's Read Me: [Color and Fonts](https://github.com/Lednerb/bilberry-hugo-theme#colors-and-fonts). 

Hope this helps!

**Comment by eosfor** [comment]
Thanks, I managed to make it

**Comment by igor-baiborodine** [comment]
@eosfor, as opposed to Issues, discussions shouldn't be closed. Therefore, I'm reopening it.
